### DISCUSSION TITLE
Preset Op - Renaming Param Doesn't Save

### DISCUSSION THREAD

**Discussion by andrejhronco** [cables-gl/cables_docs#224] [! Welcome - readme first!]
After renaming a preset param in the Preset Op, the new name doesn't save and might actually break the param preset association.

**Comment by steam0r** [comment]
hey,
i hope i understood your bug-report correctly but i was able to reproduce an issue with renaming presets, this is now fixed on dev.cables.gl. can you verify if that fixes your issue as well, please? (you might have to recreate the PresetOp as fixing a broken state is not possible, sadly).

**Reply by andrejhronco** [reply]
Hi, Still is broken unfortunately on dev as well. 
Pretty sure you understand the issue, but here are some images for further clarification:

default param name:
![image](https://user-images.githubusercontent.com/1925905/163623688-97b3a785-85a7-4651-b0bf-7e08373dddbd.png)
new name:
![image](https://user-images.githubusercontent.com/1925905/163623735-bcf8b270-f74c-4d25-8b77-37880a549cd0.png)

After save and reload, the name is still the default name.

**Reply by steam0r** [reply]
heh, then i did NOT understand your initial report and fixed another issue with renaming PRESETS...will try to reproduce and fix this

**Reply by steam0r** [reply]
i investigated this a bit further and you are right in that the new name is not saved, how does it break the param preset association though? 

do you have an example for the case where it breaks? 

i think the association is done by a unique identifier that has the old as a reference anyhow...would be great to understand this more :)

**Reply by steam0r** [reply]
that being said i fixed this behaviour on dev.cables.gl, would be nice if you could test your cases there. thanks!

**Reply by andrejhronco** [reply]
Nice, thanks for the fix, it's working well. 
The breaking occurred when updating the param names on a bunch of presets. I'll try and reproduce the issue.
